patches/0005-Modify-resources.patch

```diff
@@ -1284,21 +1284,25 @@ index 5f00d7466f..0e7f4ae71b 100644
  						},
  					},
  
-@@ -1393,7 +1395,7 @@ func resourceMsSqlDatabaseSchema() map[string]*pluginsdk.Schema {
- 						ValidateFunc: validation.StringInSlice([]string{
- 							"Disabled",
- 							"Enabled",
--						}, false),
+@@ -1426,8 +1428,10 @@ func resourceMsSqlDatabaseSchema() map[string]*pluginsdk.Schema {
+ 						Type:     pluginsdk.TypeString,
+ 						Optional: true,
+ 						Default:  EmailAccountAdminsStatusDisabled,
+-						ValidateFunc: validation.StringInSlice(PossibleValuesForEmailAccountAdminsStatus(),
+-							false),
++						ValidateFunc: validation.StringInSlice([]string{
++							"Disabled",
++							"Enabled",
```

Comment:
I'm having trouble reading this patch, but I think we are replacing `PossibleValuesForEmailAccountAdminsStatus(), false` (new upstream) with `[]string{"Disabled", "Enabled"}, features.CaseInsensitive()` and I'm not sure why. I believe that we used to be replacing `[]string{"Disabled", "Enabled"}, false` (old upstream) with `[]string{"Disabled", "Enabled"}, features.CaseInsensitive()`. Can we track upstream's change and replace with `PossibleValuesForEmailAccountAdminsStatus(), features.CaseInsensitive()`?

I have no issue believing that the answer to my question is "no", but I'd like to understand why not.

Reply:
ah! yes, the upstream changes are equivalent to our patch - `PossibleValuesForEmailAccountAdminsStatus()` contains only "Disabled" and "Enabled".

We should remove.